client/extensions/woocommerce/app/settings/payments/payment-method-bacs.js

```diff
@@ -0,0 +1,182 @@
+				buttons={ this.buttons }
+				isVisible
+			>
+				<FormFieldset className="payments__method-edit-field-container">
+					<FormLabel>{ translate( 'Title' ) }</FormLabel>
+					<FormTextInput
+						name="title"
+						onChange={ this.onEditFieldHandler }
+						value={ settings.title.value }
+					/>
+				</FormFieldset>
+				<FormFieldset className="payments__method-edit-field-container">
+					<FormLabel>{ translate( 'Instructions for customer at checkout' ) }</FormLabel>
+					<FormTextarea
+						name="description"
+						onChange={ this.onEditFieldHandler }
+						value={ method.description }
+						placeholder={ translate( 'Make your payment directly into our bank account.' ) }
+					/>
+				</FormFieldset>
+				<FormFieldset className="payments__method-edit-field-container">
+					<FormLabel>
+						{ translate( 'Instructions for customer in order email notification' ) }
+					</FormLabel>
+					<FormTextarea
+						name="instructions"
+						onChange={ this.onEditFieldHandler }
+						value={ settings.instructions.value }
+						placeholder={ translate( 'Use these bank account details…' ) }
+					/>
+				</FormFieldset>
+				<FormFieldset className="payments__method-edit-field-container">
+					<FormLegend>
+						{ translate( 'Account Details', { note: 'Fieldset legend for bank account details' } ) }
+					</FormLegend>
+					<FormFieldset className="payments__method-edit-field-container">
+						<FormLabel>{ translate( 'Account Name' ) }</FormLabel>
+						<FormTextInput
+							name="account_name"
+							onChange={ this.onEditAccountHandler }
+							value={ accountData.account_name }
+						/>
+					</FormFieldset>
+					<FormFieldset className="payments__method-edit-field-container">
+						<FormLabel>{ translate( 'Account Number' ) }</FormLabel>
+						<FormTextInput
+							name="account_number"
+							onChange={ this.onEditAccountHandler }
+							value={ accountData.account_number }
+						/>
+					</FormFieldset>
+					<FormFieldset className="payments__method-edit-field-container">
+						<FormLabel>{ translate( 'Bank Name' ) }</FormLabel>
+						<FormTextInput
+							name="bank_name"
+							onChange={ this.onEditAccountHandler }
+							value={ accountData.bank_name }
+						/>
+					</FormFieldset>
+					<FormFieldset className="payments__method-edit-field-container">
+						<FormLabel>{ translate( 'Routing Number' ) }</FormLabel>
+						<FormTextInput
+							name="sort_code"
+							onChange={ this.onEditAccountHandler }
+							value={ accountData.sort_code }
+						/>
+					</FormFieldset>
+					<FormFieldset className="payments__method-edit-field-container">
+						<FormLabel>{ translate( 'IBAN' ) }</FormLabel>
+						<FormTextInput
+							name="iban"
+							onChange={ this.onEditAccountHandler }
+							value={ accountData.iban }
+						/>
+					</FormFieldset>
+					<FormFieldset className="payments__method-edit-field-container">
+						<FormLabel>{ translate( 'BIC / Swift' ) }</FormLabel>
```

Comment:
Maybe add context here and maybe for IBAN too (i.e. at least spell out the acronyms)